app/controllers/participate/survey/index.js

```diff
@@ -1,10 +1,30 @@
 import Ember from 'ember';
+import ENV from 'isp/config/environment';
+
 
 export default Ember.Controller.extend({
   model: null,
   session: null,
-  frameIndex: 0,
-  framePage: 0,
+  frameIndex: Ember.computed(function() {
+    var frameIndex = 0;
+    if (ENV.continueSession) {
+      var session = this.get('session');
+      if (session && session.get('frameIndex')) {
+        frameIndex = session.get('frameIndex');
+      }
+    }
+    return frameIndex;
+  }),
+  framePage: Ember.computed(function() {
+    var framePage = 0;
+    if (ENV.continueSession) {
+      var session = this.get('session');
+      if (session && session.get('framePage')) {
```

Comment:
Does this mean that there is a field called `framePage` in the database record of the session? Is there a more intuitive name we might consider? 

It seems strange to have the user-facing data for the experiment named after internal implementation details. Eg a researcher looking to see how far people get in the survey wouldn't necessarily know what this key meant.